src/main/scala/configs/mllib-small.yaml

```diff
@@ -112,10 +106,44 @@ benchmarks:
     params:
       numExamples: 100
       numFeatures: 10
-  - name: classification.LinearSVC
+  - name: recommendation.ALS
     params:
       numExamples: 100
-      numFeatures: 10
+      numTestExamples: 100
+      numUsers: 100
+      numItems: 100
       regParam: 0.1
-      tol: 0.001
-      maxIter: 10
\ No newline at end of file
+      rank: 10
+      maxIter: 6
+  - name: regression.DecisionTreeRegression
+    params:
+      numExamples: 100
+      numTestExamples: 10
+      depth: 3
+      numClasses: 4
+      numFeatures: 5
+  - name: regression.GLMRegression
+    params:
+      numExamples: 100
+      numTestExamples: 10
+      numFeatures: 5
+      link: log
+      family: gaussian
+      tol: 0.0
+      maxIter: 10
+      regParam: 0.1
+  - name: regression.LinearRegression
+    params:
+      numExamples: 100
+      numTestExamples: 100
+      numFeatures: 100
+      regParam: 0.1
+      tol: [0.0]
```

Comment:
The `tol` seems should > 0 ?

Reply:
Good question -- I think if `tol` is 0, the algorithm will just run for `maxIter` iterations (LBFGS, WeightedLeastSquares, OWLQN optimizers all take `maxIter` as a constructor argument).

Reply:
I don't think it's super important what `tol` is set to here (since this config just contains small parameters to verify all of the performance tests run properly), but I'll update it to 1e-6 (the Spark default for LinearRegression).